src/test/java/teammates/test/cases/browsertests/AdminHomePageUiTest.java

```diff
@@ -74,14 +74,24 @@ private void testCreateInstructorAction() throws Exception {
         InstructorAttributes instructor = new InstructorAttributes();
         
         String shortName = "Instrúctör";
-        instructor.name = "AHPUiT Instrúctör";
-        instructor.email = "AHPUiT.instr1!@gmail.tmt";
+        instructor.name = "AHPUiT Instrúctör+++";
+        instructor.email = "AHPUiT+++_.instr1!@gmail.tmt";
         String institute = "TEAMMATES Test Institute 1";
-        String demoCourseId = "AHPUiT.instr1_.gma-demo";
+        String demoCourseId = "AHPUiT____.instr1_.gma-demo";
         
         String instructorDetails = instructor.name + " | " + instructor.email + "\n"
                                  + instructor.name + " | " + instructor.email + " | " + institute;
         
+        ______TS("action success: instructor details are sent across to the backend properly");
+        BackDoor.deleteAccount(TestProperties.TEST_INSTRUCTOR_ACCOUNT);
+        BackDoor.deleteCourse(demoCourseId);
+        BackDoor.deleteInstructor(demoCourseId, instructor.email);
+        homePage.createInstructorByInstructorDetailsSingleLineForm(instructorDetails);
```

Comment:
Oh yea you're right. I initially used a different set of instructor details for this test alone, so I had to do this. Then I realised that it wasn't necessary to use a different set of details and removed those, but forgot to remove this. I'll fix it. Thanks for pointing it out.